resources/fcontext.rb

```diff
@@ -5,3 +5,4 @@
 default_action :addormodify
 attribute :file_spec, :kind_of => String, :name_attribute => true
 attribute :secontext, :kind_of => String
+attribute :filetype, :kind_of => String, :equal_to => ['a', 'f', 'd', 'c', 'b', 's', 'l', 'p']
```

Comment:
Why? I don't see any added benefit in it, defaulting to not using the parameter makes sure it will always work on any platform. Defaulting to this could break support or require additional checks at some point.